terraform/files/bin/apply_openstack_integration.sh

```diff
@@ -24,14 +24,26 @@ else
 fi
 echo "# Install external OpenStack cloud provider $OCCM_VERSION to $CLUSTER_NAME"
 
+OCCM_RAW=$OCCM_VERSION
+OCCM_OVN=v1.26.202
+if grep '^lb\-provider=ovn' ~/$CLUSTER_NAME/cloud.conf >/dev/null 2>&1; then
+  OCCM_VERSION=$OCCM_OVN
+  OCCM_RAW=v1.26.2
```

Comment:
What about version compatibility, will it also work for Kubernetes < 1.26? E.g. we use CCMR_VERSION(roles, role-bindings) below, but it can be in a different minor version.

Reply:
Hi Roman,
we are prudent to use the recommended OCCM versions that match the k8s version. That said, I have not yet been able to observe incompatibility issues with newer OCCMs since the 1.20 times or so ...
I have tested the 1.26 OCCM successfully with 1.25 clusters.
The feature is experimental anyhow for now (until we can get a patch merged upstream), maybe we should just add a note to the documentation that the customer OCCM build has only been tested with k8s-1.25 and -1.26.

Care to do some ingress testing on gx-scs with this?

Reply:
Thank you for that note in the documentation, I will do ingress testing.

Reply:
Ok, I did some testing with a simple app with ingress in front of it, and it doesn't work as expected for me. It seems like the health monitor doesn't work because sometimes I get `Connection timed out` when the request is going from outside of the cluster. From within the cluster, it works.

My setup is the default test cluster, with `USE_OVN_LB_PROVIDER: true` and `--allow-preview-features`.
I am not able to investigate if it is really a health monitor issue or not. But it seems like 2/3 of requests are not balanced correctly. But in the Horizon everything looks ok, only one node is online.

Reply:
From the upstream, I can see that the fix is probably included in the 3.1.0 version of openstack/ovn-octavia-provider project but I don't know if it is included in the latest stable zed, because [here](https://docs.openstack.org/releasenotes/ovn-octavia-provider/zed.html) is only 3.0.0 mentioned. But [here](https://releases.openstack.org/zed/#zed-ovn-octavia-provider) the 3.1.0 is mentioned as the most recent version and also [here](https://docs.openstack.org/ovn-octavia-provider/zed/) is 3.1.x.

Reply:
Christian @berendt, what is the best way to find out what version of octavia-ovn-provider is in OSISM-5.1.0?

Reply:
@chess-knight @jschoone - The question is whether we should merge this change despite upstream OVN provider not being fully ready - preparing for the time when it becomes ready and allowing to test and expose the issue.
Of course, we also need to then push for the patch (or a variant of it that is configurable?) to go into upstream OCCM.

Reply:
> Christian @berendt, what is the best way to find out what version of octavia-ovn-provider is in OSISM-5.1.0?

The answer is researched in https://github.com/osism/issues/issues/499.